modules/admin-ui-picard/app/src/components/events/partials/modals/SeriesDetailsModal.js

```diff
@@ -1,15 +1,27 @@
 import React from "react";
 import { useTranslation } from "react-i18next";
 import SeriesDetails from "./SeriesDetails";
+import { hasChanges } from "../../../shared/modals/ResourceDetailsAccessPolicyTab";
 
 /**
  * This component renders the modal for displaying series details
  */
 const SeriesDetailsModal = ({ handleClose, seriesTitle, seriesId }) => {
 	const { t } = useTranslation();
 
+	let confirmUnsaved = () => {
+		return window.confirm(t("CONFIRMATIONS.WARNINGS.UNSAVED_CHANGES"));
+	};
+
 	const close = () => {
-		handleClose();
+		if (hasChanges) {
+			if (confirmUnsaved()) {
```

Comment:
Can simplify this to

```suggestion
	if (hasChanges && confirmUnsaved()) {
```

due to the short-circuiting behavior of the && operator .